### DISCUSSION TITLE
Problem z uslugami na serwerze, sklep nie dziala

### DISCUSSION THREAD

**Discussion by DzikusCorporation** [gammerce/sklep-sms#305] [General]
Witam mam problem z odpaleniem sklepu na serwerze 

otoz te 3 uslugi sa w takim stanie:

 [ 78] Shop SMS: VIP           3.11.3      SeeK              shopsms_service  stopped  
 [ 79] Shop SMS: VIP           3.11.3      SeeK              shopsms_service  stopped  
 [ 80] Shop SMS: VIP           3.11.3      SeeK              shopsms_service  stopped  

licencja dziala:

[  1] Shop SMS                3.11.3      SeeK              shopsms.amxx     running  

a pisze na serwerze po wpisaniu /sklepsms , ze na ma uslug do kupna czy cos 

[SKLEP-SMS] No services are available for purchase.

Nie działa to ... proszę o zwrot pięniedzy

**Comment by budziam** [comment]
Cześć,
w panelu administratora sklepu trzeba włączyć odpowiednie usługi na konkretnym serwerze. Domyślnie wszystkie usługi są wyłączone. Usługę można włączyć edytując usługę, albo edytując serwer przez panel admina.
